.travis.yml

```diff
@@ -4,46 +4,6 @@ sudo: required
 
 matrix:
     include:
-        - name: "Windows, Python Latest: multiprocessing"
```

Comment:
I guess we should look into if anyone has had this problem before we drop all this.